swagger.json

```diff
@@ -0,0 +1,1344 @@
+{
+  "consumes": [
+    "application/json",
+    "application/x-protobuf"
+  ],
+  "produces": [
+    "application/json",
+    "application/x-protobuf"
+  ],
+  "schemes": [
+    "http",
+    "https"
+  ],
+  "swagger": "2.0",
+  "info": {
+    "description": "Snap is an open telemetry framework designed to simplify the collection,\nprocessing and publishing of system data through a single API.\n\nThis specification should demonstrate a fully compliant swagger 2.0 spec for Snap.\nIt lists all the possible Snap Restful APIs that are available for use,\ntesting or generating spec compatible clients.",
+    "title": "Snap Restful APIs.",
+    "termsOfService": "There are no TOS at this moment, use at your own risk we take no responsibility.",
+    "contact": {
+      "name": "Snap Maintainer",
+      "url": "https://github.com/intelsdi-x/snap",
+      "email": "snap@intelsdi-x.slack.com"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/"
+    },
+    "version": "1.1.0"
```

Comment:
I have a  question about the version - it looks like the version of Snap (however the latest one is 1.2.0, not 1.1.0). How should it be managed when a new release is coming? Or maybe it's rather about version of Snap Restful API? Could you explain me that? Thank You